### PR TITLE
First draft of final exercise for GitHub workflows (chapter 11)

### PR DESCRIPTION
**Pull request by lwjohnst86**
@k8hertweck I moved an item over from your Git as solo chapter, since it seemed to make more sense here.

### CODE REVIEW

File: chapters/11-version-control-workflow.Rmd [hunk 1]
```diff
@@ -2,12 +2,28 @@
...
+- E.g. with pr_* functions from usethis
 - Using this workflow as a team or as a contributor
 
-
-
 ## Final exercise
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+Complete these tasks for your package for the first project assignment.
```
Comment (ian-flores) [inline chapters/11-version-control-workflow.Rmd:12]: "Complete these tasks as part of the package for your first project assignment" would be my suggestion on how to phrase this.

File: chapters/11-version-control-workflow.Rmd [hunk 2]
```diff
@@ -2,12 +2,28 @@
...
+`pr_init("fixing-function")`.
+3. Make the following changes to the FIXME function:
+    - FIXME
+4. Add and commit these changes.
+5. Push the changes up to GitHub by using `pr_push()`.
+    - Create a new Pull Request based on the push.
+    - Merge the newly created Pull Request.
+6. In your local Git repository, run `pr_finish()` to finish up.
+
+Repeat this process by a few more times by making changes to:
```
Comment (ian-flores) [inline chapters/11-version-control-workflow.Rmd:26]: How many times is "a few more times"? I would predict that the educational value of the activity doesn't increase much after one or at most two iterations?

**Review by ian-flores — APPROVED** [review]
I made some small suggestions but overall looks good to me.

File: chapters/11-version-control-workflow.Rmd (hunk 2 above)
Comment (lwjohnst86) [reply chapters/11-version-control-workflow.Rmd:26]: Yea, maybe one more time would help solidify the action. Changed :)

**Pull request merged**